gamemodes/terrortown/gamemode/server/sv_roleselection.lua

```diff
@@ -372,17 +590,17 @@ end
 -- @note The subrole replaces a previously selected baserole.
 --
 -- @param table plys The players that should receive roles.
--- @param table roleData The @{ROLE} object of the considered role.
+-- @param int subrole The @{ROLE} index of the considered role.
```

Comment:
Is there an `int` type?

Reply:
Done with https://github.com/TTT-2/TTT2/pull/649/commits/f26eb2e183979dfd5d55f2660101db70d14b766c